app/assets/stylesheets/header.scss

```diff
@@ -33,10 +33,12 @@ $header_background_image: 'ADRL_BG_1280x160.png';
 
 #header-navbar .navbar-right .login-link p,
 .page-header-splash .login-link p {
+  color: $login_link_color;
```

Comment:
Why are you changing this? this seems to make the plain text look like a link? How do you differentiate a link from text?